public/app/features/org/__snapshots__/OrgDetailsPage.test.tsx.snap

```diff
@@ -43,6 +43,7 @@ exports[`Render should render organization and preferences 1`] = `
         orgName="Cool org"
       />
       <SharedPreferences
+        disabled={false}
```

Comment:
Does it make sense to leave a short comment here explaining why this is always visible?

Reply:
This is the snapshot for test comparison. Do you think it's important?

Reply:
No, my miss. Have not realised that. Thanks for the comment.